### DISCUSSION TITLE
Family unknown

### DISCUSSION THREAD

**Discussion by whgibbo** [aquasecurity/trivy#7563] [Q&A]
### Question

Is there a command line to set the family / platform for a docker image..  As currently it is not detecting the family.
It is based on ALMA 9.4.

I noticed there is --platform and have tried passing:
`trivy image --platform alma/amd64 <imagename>`

But still comes back with:
```
INFO    Detected OS     family="none" version=""
WARN    Unsupported os  family="none"
```

Any suggestions for the command line or changes I need to make to the /etc/os-release file?

### Target

Container Image

### Scanner

None

### Output Format

None

### Mode

Standalone

### Operating System

Ubuntu

### Version

```bash
Version: 0.55.2
Vulnerability DB:
  Version: 2
  UpdatedAt: 2024-09-20 06:12:54.353543232 +0000 UTC
  NextUpdate: 2024-09-20 12:12:54.353542941 +0000 UTC
  DownloadedAt: 2024-09-20 08:21:59.08789703 +0000 UTC
```

**Comment by DmitriyLewen** [comment]
Hello @whgibbo 

We parse `etc/almalinux-release` file to detect Alma Linux - https://github.com/aquasecurity/trivy/blob/main/pkg/fanal/analyzer/os/redhatbase/alma.go

With official image Trivy works correctly:
```
➜  trivy image almalinux:9.4
...
2024-09-24T16:54:00+06:00	INFO	Detected OS	family="alma" version="9.4"
2024-09-24T16:54:00+06:00	INFO	[alma] Detecting vulnerabilities...	os_version="9" pkg_num=151
...
```

> I noticed there is --platform and have tried passing: 

This is analogous to the `platform` flag in Docker.

Regards, Dmitriy
